beacon-chain/slasher/iface/interface.go

```diff
@@ -0,0 +1,15 @@
+// Package iface defines the interface for interacting with the slasher,
```

Comment:
We don't need this separate package. We do this in other places because typically there is some circular dependency and we need the interface to be in its own package. You can just move `type SlashingChecker interface` right above type Service struct in the slasher package

Reply:
Also, yay for inversion of control! Now you know this awesome pattern in Go :)